lib/LANraragi/Model/Config.pm

```diff
@@ -40,6 +40,10 @@ sub get_redis {
         reconnect => 3
     );
 
+    if ($ENV{LANDATADIR}) {
+        $redis->config_set("dir", $ENV{LANDATADIR});
```

Comment:
Since you're setting dir in the redis config file, that part is now unnecessary.

Reply:
removed that part now, I missed it in the commits before.